src/DataLib/fendl32B_retrofit/gendf_tools.py

```diff
@@ -0,0 +1,201 @@
+# Import packages
+import argparse
+import csv
+import sys
+from logging_config import logger, LoggerWriter
+import ENDFtk
+import contextlib
+import os
+
+# Define an argument parser
+def fendl_args():
+    # Temporarily reset stdout and stderr to the defaults
+    # so that arg messages (i.e. --help) print out to terminal
+    original_stdout = sys.stdout
+    original_stderr = sys.stderr
+
+    try:
+        sys.stdout = sys.__stdout__
+        sys.stderr = sys.__stderr__
```

Comment:
Why do you need this?  It seems pretty unusual for an `argparse` setup.  I've never seen this before

Reply:
I tried commenting this out, but it had the effect of sending the `argparse` help messages to the logger, rather than the terminal, which seems less helpful or desirable than having it print to the terminal, which it currently does.